CHANGELOG.rst

```diff
@@ -2,30 +2,200 @@
 Release Notes
 =============
 
-.. _Release Notes_7.22.1:
```

Comment:
I don't think we should delete the 7.22.1 release notes, should we?

Reply:
Not sure why it got deleted tbh, I ran 

```inv release.update-changelog <new v7 version>```

in the release branch to generate this. Maybe because it's a bugfix release?

Reply:
I am not sure either. `7.21.1` changelog is still there so I don't really know. Maybe @KSerrania might shed some light into this?

Reply:
That's because the `7.22.1` changelog isn't present in the `7.23.x` branch, due to it being merged to master after `7.23.x` was branched out (which, in turn, was caused by the fact that `7.22.1` was released during the `7.23.0` freeze, which is something pretty uncommon).

The `7.23.0` changelog should indeed be added on top of the `7.22.1` one, not replacing it.

Reply:
I've readded it manually now. Thanks for the pointers guys!